Update members list and add heading for Performance Points Maintainers on `People/Performance_Points_Committee`

tsunyoku opened the pull request:
- Adds a section for performance points maintainers
- Makes adjustments to performance points committee page to better reflect workflow
- Moves recently retired performance points committee members to previous members

stanriders commented on wiki/People/Performance_Points_Maintainers/en.md:
```diff
@@ -0,0 +1,27 @@
+<!-- internal reference: https://github.com/orgs/ppy/teams/pp-maintainers -->
+
+| osu! profile | GitHub profile |
+| :-- | :-- | :-- |
```
```suggestion
| :-- | :-- |
```

Walavouchey reviewed (commented):
i question the need for this new article. it's just 2 people that are already in the pp committee

if you want to document the group that can merge into `pp-dev`, you can add that as a new section in the pp committee article, mirroring how `ppy/news-approval` is a separate section in the [wiki maintainer list](https://osu.ppy.sh/wiki/en/People/osu%21_wiki_maintainers)

tsunyoku commented:
Sorry, took me a while to get back to this one with the deploy and everything else going on. I've deleted the separate page and added a heading with a brief description alongside the member list. Hopefully this is better!

Hiviexd commented on wiki/People/Performance_Points_Committee/en.md:
```diff
@@ -5,26 +5,35 @@ tags:
-## Members
+## Performance Points Maintainers
+
+The maintainers are hand-picked by osu! developers and consist of people with a lot of experience organising performance points efforts. They are responsible for writing newsposts, merging batches of changes into the `pp-dev` branch on the osu! repository and acting as a final approval on all PP changes.
```
```suggestion
The maintainers are hand-picked by osu! developers and consist of people with a lot of experience organising performance points efforts. They are responsible for writing news posts, merging batches of changes into the [`pp-dev`](https://github.com/ppy/osu/tree/pp-dev) branch on the [osu! repository](https://github.com/ppy/osu) and acting as a final approval on all PP changes.
```
